core/app/assets/javascripts/refinery/admin.js.erb

```diff
@@ -591,15 +598,23 @@ var page_options = {
       $('#new_page_part_dialog').dialog('open');
     });
 
+    $('#new_page_part_title').on('keyup', function() {
```

Comment:
This should be on "input" event. But it doesn't work for IE8 and IE9. But alternatively "propertychange" event can be used there.